examples/pgadmin/pgadmin.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PGAdmin
+metadata:
+  name: pgadmin
+spec:
+  image: registry.developers.crunchydata.com/crunchydata/crunchy-pgadmin4:ubi8-4.30-17
```

Comment:
if we do want an example, maybe we move towards using related images so we don't have to keep it updated in multiple places :thinking:

Reply:
:+1: That's a good idea I think. I'll work on implementing that.

Reply:
To follow up, forgot I already implemented that function. We should be good, I just need to remove the `image:` line :upside_down_face: